cmd/spiffe-helper/config/config.go

```diff
@@ -76,7 +81,7 @@ func ParseConfig(file string) (*Config, error) {
 }
 
 // ParseConfigFlagOverrides handles command line arguments that override config file settings
-func (c *Config) ParseConfigFlagOverrides(daemonModeFlag bool, daemonModeFlagName string) {
+func (c *Config) ParseConfigFlagOverrides(daemonModeFlag bool) {
```

Comment:
Can you restore the functionality where the flag name is passed in here? And the `DaemonModeFlagName` const is duplicated in config_test.go rather than in config.go? If its going to be duplicated Id prefer the duplicate to be in the tests.